misk/src/main/kotlin/misk/security/cert/X500Name.kt

```diff
@@ -1,16 +1,21 @@
 package misk.security.cert
 
-class X500Name(private val components: Map<String, String>) {
-  val commonName = get("CN")
-  val organization = get("O")
-  val organizationalUnit = get("OU")
-  val state = get("ST")
-  val locality = get("L")
-  val country = get("C")
-
-  operator fun get(componentName: String) = components[componentName.toUpperCase()]
-
-  fun asMap() = components
+data class X500Name(
+  val commonName: String?,
```

Comment:
I wonder if these fields would be better as `cn`, `ou`, `o`, `l`, `st`, `c`. The words are almost irrelevant!

Reply:
or perhaps the uppercase form, YOLO, `CN`, `OU`, `O` etc.!